### PR TITLE
Closes #914 (SympTEMIST)

### LINKED ISSUE #914: SympTEMIST dataset
## Adding a Dataset
- **Name:** *SympTEMIST*
- **Description:** *BioCreative VIII SympTEMIST task https://temu.bsc.es/symptemist/*
- **Task:** *NER, NED*
- **Paper:** *https://zenodo.org/records/10104547*
- **Data:** *https://doi.org/10.5281/zenodo.8223653*
- **License:** *[Creative Commons Attribution 4.0 International License](http://creativecommons.org/licenses/by/4.0/).*
- **Motivation:** *High-quality gold-standard similar to SympTEMIST, CANTEMIST, ...*

### PR DESCRIPTION
**Pull request by phlobo**
closes #914

### Checkbox

- [x] Confirm that this PR is linked to the dataset issue.
- [x] Create the dataloader script `hub/hub_repos/my_dataset/my_dataset.py` (please use only lowercase and underscore for dataset naming).
- [x] Provide values for the `_CITATION`, `_DATASETNAME`, `_DESCRIPTION`, `_HOMEPAGE`, `_LICENSE`, `_URLs`, `_SUPPORTED_TASKS`, `_SOURCE_VERSION`, and `_BIGBIO_VERSION` variables.
- [x] Implement `_info()`, `_split_generators()` and `_generate_examples()` in dataloader script.
- [x] Make sure that the `BUILDER_CONFIGS` class attribute is a list with at least one `BigBioConfig` for the source schema and one for a bigbio schema.
- [x] Confirm dataloader script works with `datasets.load_dataset` function.
- [x] Confirm that your dataloader script passes the test suite run with `python -m tests.test_bigbio_hub <dataset_name> [--data_dir /path/to/local/data] --test_local`.
- [x] If my dataset is local, I have provided an output of the unit-tests in the PR (please copy paste). This is OPTIONAL for public datasets, as we can test these without access to the data files.

### CODE REVIEW

File: bigbio/hub/hub_repos/symptemist/symptemist.py [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+
+_HOMEPAGE = "https://temu.bsc.es/symptemist/"
+
+_LICENSE = "CC_BY_4p0"
+
+_URLS = {
+    _DATASETNAME: "https://zenodo.org/records/10635215/files/symptemist-complete_240208.zip?download=1",
+}
+
+_SUPPORTED_TASKS = [Tasks.NAMED_ENTITY_RECOGNITION, Tasks.NAMED_ENTITY_DISAMBIGUATION]
```
Comment (leonweber) [inline bigbio/hub/hub_repos/symptemist/symptemist.py:53]: `NAMED_ENTITY_DISAMBIGUATION` is in supported tasks, but the tests report that there are no normalization entries (see stacktrace below). @phlobo is this intended?

```
======================================================================
FAIL: runTest (__main__.TestDataLoader.runTest) [Check schema validity]
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/lwg/projects/biomedical/tests/test_bigbio_hub.py", line 130, in runTest
    self.test_schema(schema)
  File "/Users/lwg/projects/biomedical/tests/test_bigbio_hub.py", line 761, in test_schema
    raise AssertionError(f"Required key '{non_empty_feature}' does not have any instances")
AssertionError: Required key 'normalized' does not have any instances
```

File: bigbio/hub/hub_repos/symptemist/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  - NAMED_ENTITY_RECOGNITION
+  - NAMED_ENTITY_DISAMBIGUATION
+---
+
+
+# Dataset Card for SympTEMIST
+
+## Dataset Description
+
+- **Homepage:** https://temu.bsc.es/symptemist/
```
Comment (leonweber) [inline bigbio/hub/hub_repos/symptemist/README.md:23]: The entity and document statistics reported on the webpage (https://temu.bsc.es/symptemist/corpus-description/) do not match those reported by the tests (see below). Is this expected?

```
train
==========
id: 304
document_id: 304
passages: 304
entities: 3484
normalized: 3484
events: 0
coreferences: 0
relations: 0

test
==========
id: 246
document_id: 246
passages: 246
entities: 2848
normalized: 2848
events: 0
coreferences: 0
relations: 0
```

**Review by leonweber — CHANGES_REQUESTED** [review]
Thanks for the PR @phlobo. Could you please have a look at the comments?

File: bigbio/hub/hub_repos/symptemist/symptemist.py (hunk 1 above)
Comment (phlobo) [reply bigbio/hub/hub_repos/symptemist/symptemist.py:53]: There were multiple tracks (one with entities only and another one with normalization). We had the same issue with DisTEMIST: https://github.com/bigscience-workshop/biomedical/pull/736#discussion_r934932383

Comment (leonweber) [reply bigbio/hub/hub_repos/symptemist/symptemist.py:53]: Thanks for clarifying!

File: bigbio/hub/hub_repos/symptemist/README.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+---
+language:
+  - es
+bigbio_language:
+  - Spanish
+license: Creative Commons Attribution 4.0 International License
```
Comment (leonweber) [inline bigbio/hub/hub_repos/symptemist/README.md:6]: When trying to upload to the hub I'm getting the following error. Could you please update the license field?

```
Bad request:
"license" must be one of [apache-2.0, mit, openrail, bigscience-openrail-m, creativeml-openrail-m, bigscience-bloom-rail-1.0, bigcode-openrail-m, afl-3.0, artistic-2.0, bsl-1.0, bsd, bsd-2-clause, bsd-3-clause, bsd-3-clause-clear, c-uda, cc, cc0-1.0, cc-by-2.0, cc-by-2.5, cc-by-3.0, cc-by-4.0, cc-by-sa-3.0, cc-by-sa-4.0, cc-by-nc-2.0, cc-by-nc-3.0, cc-by-nc-4.0, cc-by-nd-4.0, cc-by-nc-nd-3.0, cc-by-nc-nd-4.0, cc-by-nc-sa-2.0, cc-by-nc-sa-3.0, cc-by-nc-sa-4.0, cdla-sharing-1.0, cdla-permissive-1.0, cdla-permissive-2.0, wtfpl, ecl-2.0, epl-1.0, epl-2.0, etalab-2.0, eupl-1.1, agpl-3.0, gfdl, gpl, gpl-2.0, gpl-3.0, lgpl, lgpl-2.1, lgpl-3.0, isc, lppl-1.3c, ms-pl, apple-ascl, mpl-2.0, odc-by, odbl, openrail++, osl-3.0, postgresql, ofl-1.1, ncsa, unlicense, zlib, pddl, lgpl-lr, deepfloyd-if-license, llama2, llama3, gemma, unknown, other, array]
```

**Review by leonweber — CHANGES_REQUESTED** [review]
Thanks for the clarifications @phlobo. I have one more request regarding the license.

File: bigbio/hub/hub_repos/symptemist/README.md (hunk 3 above)
Comment (phlobo) [reply bigbio/hub/hub_repos/symptemist/README.md]: I fixed the Readme, can you try again?

**Review by leonweber — APPROVED** [review]
LGTM. Thanks a lot!

**Review by leonweber — APPROVED** [review]

File: bigbio/hub/hub_repos/symptemist/README.md (hunk 3 above)
Comment (leonweber) [reply bigbio/hub/hub_repos/symptemist/README.md]: @phlobo Merged and uploaded. Thanks : )!

**Pull request merged**